src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -122,24 +124,24 @@ public async Task UpdateAssetsAsync(
         Guid subscriptionId,
         SubscriptionType type,
         int buildId,
-        string sourceRepo,
-        string sourceSha,
-        List<Asset> assets,
         bool forceApply)
     {
+        _logger.LogInformation("Looking up build {buildId}", buildId);
```

Comment:
I think we agreed we don't want to log outside of methods that do something about what they are going to do but instead we would log inside.
I think SQL queries are already logged (with a higher verbosity though) so maybe these "looking up build" statements are extraneous.

Reply:
I think they're currently not included in the AI, we might have disabled them in the settings.json. But even if we had them, those are quite hard to read, and just give you a translated SQL query, maybe even without the parameters, I'd prefer if we had these simple traces